sdk/include/unwind.h

```diff
@@ -25,11 +25,8 @@ struct CleanupList
  */
 __always_inline static inline struct CleanupList **cleanup_list_head()
 {
-	void     *csp = __builtin_cheri_stack_get();
-	ptraddr_t top = __builtin_cheri_top_get(csp);
-	csp           = __builtin_cheri_address_set(
-	            csp, top - INVOCATION_LOCAL_UNWIND_LIST_OFFSET);
-	return (struct CleanupList **)csp;
+	return (struct CleanupList **)invocation_state_slot(
+	  INVOCATION_LOCAL_UNWIND_LIST_OFFSET / sizeof(void *) - 1);
```

Comment:
This seems like a very convoluted way of spelling `0`. I had to look up the definitions of both `invocation_state_slot` and `INVOCATION_LOCAL_UNWIND_LIST_OFFSET` to figure out what's going on. Maybe hide it in a macro `INVOCATION_LOCAL_UNWIND_SLOT` with a suitable comment?

Reply:
How's that (a `_Static_assert`)?